src/components/pages/Login/loginValidation.js

```diff
@@ -0,0 +1,37 @@
+import * as yup from 'yup';
+
+
+// login schema
+const loginSchema = yup.object().shape({
+  username: yup.string().min(5).required(),
+  password: yup.string().min(5).required()
```

Comment:
For the back-end, password should have length of at least 6.

Reply:
Fixed, good catch! 👍